collect_app/src/main/java/org/odk/collect/android/tasks/TimerSaveTask.java

```diff
@@ -15,7 +15,7 @@
  * Background task for appending a timer event to the timer log
  */
 public class TimerSaveTask extends AsyncTask<TimerLogger.Event, Void, Void> {
-    private static File file;
+    private File file;
     private static final String TIMING_CSV_HEADER = "event, node, start, end";
 
     public TimerSaveTask(File file) {
```

Comment:
If a null File is an error, could we mark this `@NonNull` and handle the null case at use? It looks like this is only being used in one place currently.